src/stores/store.ts

```diff
@@ -359,14 +369,14 @@ export const useStore = defineStore('store', {
         });
 
         // TODO: consider moving logging to client.ts
-        util.infoLogger('PLOTTING PROGRESS | block subscription started');
+        util.infoLogger('block subscription started');
       } catch (error) {
         // TODO: consider moving logging to client.ts
-        util.errorLogger('PLOTTING PROGRESS | Farmer start error!');
+        util.errorLogger('Sync error: ' + error as string);
         this.setError({
-          title: 'errorPage.startFarmerFailed',
+          title: 'errorPage.startFarmerFailed', // TODO: make a page for sync error
```

Comment:
We don't need another page, currently, we have one component for all errors with different titles and messages

Reply:
I would suggest we don't make these changes in `store.ts` unless we clarify error resolutions #308